src/NuGet.Core/NuGet.Common/Strings.resx

```diff
@@ -147,9 +147,14 @@
   </data>
   <data name="UnsupportedHashAlgorithm" xml:space="preserve">
     <value>Hash algorithm '{0}' is unsupported. Supported algorithms include: SHA512 and SHA256.</value>
```

Comment:
should we also update this string to have the list of supported algorithms passed as an argument?

Reply:
The string is only used [here](https://github.com/NuGet/NuGet.Client/blob/7a8fe0bd7397e28a4fda6479b49e22df76578ece/src/NuGet.Core/NuGet.Common/CryptoHashProvider.cs#L50). THe method only accepts SHA256 and SHA512.